CONTRIBUTING.md

```diff
@@ -124,7 +124,7 @@ provide as much detail and context as possible.
 
 ## Pull requests
 
-Good pull requests—patches, improvements, new features—are a fantastic
+Good pull requests - patches, improvements, and new features - are a fantastic
```

Comment:
We use a number of em dashes throughout our docs I believe—let's leave it as-is for now for consistency with what's written most other places.